runhouse/resources/module.py

```diff
@@ -603,6 +604,48 @@ def refresh(self):
         else:
             return self
 
+    def replicate(self, num_replicas=1, names="auto", envs="auto"):
```

Comment:
does `num_replicas` include the current?

Reply:
Yes. We can switch it or make it an option if a users want that.